### PR TITLE
gupnp 1.2.4

### PR DESCRIPTION
**Pull request by tschoonj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SeekingMeaning** [comment]
CI issue should be fixed by Homebrew/homebrew-test-bot#479

File: Formula/gupnp.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Gupnp < Formula
 
   desc "Framework for creating UPnP devices and control points"
   homepage "https://wiki.gnome.org/Projects/GUPnP"
-  url "https://download.gnome.org/sources/gupnp/1.2/gupnp-1.2.3.tar.xz"
-  sha256 "d447e54d88e4a8fab84ad1766070e9208e21166fc7e2ce95df6e33e49e8d29b1"
-  revision 1
+  url "https://download.gnome.org/sources/gupnp/1.2/gupnp-1.2.4.tar.xz"
+  sha256 "f7a0307ea51f5e44d1b832f493dd9045444a3a4e211ef85dfd9aa5dd6eaea7d1"
+  license "LGPL-2.0"
```
Comment (SeekingMeaning) [inline Formula/gupnp.rb:8]:
```suggestion
  license "LGPL-2.0-or-later"
```

**Comment by chenrui333** [comment]
maybe squash the commits?

**Comment by tschoonj** [comment]
> maybe squash the commits?

Done!

**Review by SeekingMeaning — APPROVED** [review]
Thanks, @tschoonj!

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**